plugins/hls-tactics-plugin/src/Ide/Plugin/Tactic/LanguageServer/TacticProviders.hs

```diff
@@ -71,6 +75,25 @@ commandProvider HomomorphismLambdaCase =
   requireExtension LambdaCase $
     filterGoalType ((== Just True) . lambdaCaseable) $
       provide HomomorphismLambdaCase ""
+commandProvider UseDataCon =
+  requireFeature FeatureUseDataCon $
+    filterTypeProjection
+        ( guardLength (<= 5)
```

Comment:
`5` should be configurable

Reply:
Do you think anyone would actually ever change this value, though?